web/EXAMPLES.md

```diff
@@ -24,15 +24,20 @@ For each third party asset request on the site, take the hostname of a request,
         - If a rule match is found, then for the first matching rule:
 
             a. If the rule has `action: ignore`, do not block[^2]
+	    
+	      b. If there are rule `options`, do any of the domains and request types match this request?[^1]
 
-            b. If there are rule `exceptions`, do any of the domains and request types match this request?[^1]
+	      - No: Continue to next rule
+	      - Yes: Continue to next step (C below)
+
+          c. If there are rule `exceptions`, do any of the domains and request types match this request?[^1]
 
 	      - Yes: Don't block[^2]
 	      - No: Block
```

Comment:
so if there are exceptions `surrogate` will never be used?

Reply:
Correct